src/components/Menu.jsx

```diff
@@ -12,6 +12,8 @@ const propTypes = {
   renderItemLabel: PropTypes.func,
   value: PropTypes.string,
   onSelect: PropTypes.func,
+  shouldTransferFocus: PropTypes.bool,
+  notifyBeforeFocusWrap: PropTypes.func,
 };
```

Comment:
Also, how would we go about enforcing that the callback function passed in as a prop from the parent returns a number or null in order to set the state of the focusedIndex? Is this something that is enforceable outside of a simple type check within handleKeyDown?